.github/workflows/publish-kata-deploy-payload.yaml

```diff
@@ -63,7 +63,6 @@ jobs:
           sudo rm -rf /usr/share/dotnet
           sudo rm -rf /opt/ghc
           sudo rm -rf /usr/local/share/boost
-          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
```

Comment:
I was wondering where this variable is defined and used. I can see only 3 hits where the removal is executed: https://github.com/search?q=repo%3Akata-containers%2Fkata-containers%20AGENT_TOOLSDIRECTORY&type=code